cmd/gtk/utils.go

```diff
@@ -181,7 +182,11 @@ func updateFeeHint(lbl *gtk.Label, amtStr string, w *wallet.Wallet) {
 	if err != nil {
 		updateHintLabel(lbl, "")
 	} else {
-		fee := w.CalculateFee(amount)
+		fee, err := w.CalculateFee(amount, payload.PayloadTypeTransfer)
```

Comment:
This only works for Transfer type.
To ensure the system works fine with Non-transfer payloads, it's better to pass the payload type as an argument to this function.